### DISCUSSION TITLE
how to use webdav for storage backend?

### DISCUSSION THREAD

**Discussion by gdtv** [rustic-rs/rustic#1179] [Q&A] [answered]
Hello everyone, I am a new user. I saw here https://rustic.cli.rs/docs/comparison-restic.html that Rustic supports WebDAV, OneDrive, etc. as storage backends. But I didn't find how to set up webdav in the "Preparing a new repository" section in the document https://rustic.cli.rs/docs/commands/init/intro.html. Does it mean I need to use rclone as a bridge to access webdav, onedrive, etc?

**Comment by kapitainsky** [comment, marked as answer]
Documentation seems always to be behind vs. development progress:) 

Rustic uses [opendal](https://opendal.apache.org/docs/rust/opendal/services/index.html) to facilitate access to various remotes out of the box - including webdav. Here you are example how to configure it:

https://github.com/rustic-rs/rustic/blob/main/config/services/webdav_owncloud_nextcloud.toml

Of course there is always possibility to use rclone as well.
